### PR TITLE
[Example] UI update for PoseNet

### LINKED ISSUE #476: [Example] Code refactor for PoseNet

### PR DESCRIPTION
**Pull request by ibelem**
Fix #476
Please refer to [PoseNet Test URL](https://ibelem.github.io/webml-website/examples/posenet/index.html)

### CODE REVIEW

**Comment by ibelem** [comment]
@pinzhenx @huningxin @Christywl @BruceDai Please PTAL, thanks!

File: examples/third_party/dat.gui.js [hunk 1]
```diff
@@ -573,1958 +561,1950 @@ var inherits = function (subClass, superClass) {
...
+          }, this);
+          if (mismatch) {
+            Common.extend(this.__color.__state, i);
           }
-        }, this);
-        if (mismatch) {
-          Common.extend(this.__color.__state, i);
-        }
-      }
-      Common.extend(this.__temp.__state, this.__color.
```
Comment (pinzhenx) [inline examples/third_party/dat.gui.js]: syntax error

File: examples/posenet/utils.js [hunk 2]
```diff
@@ -133,17 +128,15 @@ class Utils{
...
     this.displacementFwd = new Float32Array(this.DISPLACEMENT_FWD_SIZE);
     this.displacementBwd = new Float32Array(this.DISPLACEMENT_BWD_SIZE);
-    this.model = new PoseNet(this.modelArch, Number(this._version), this._use_atrous_conv, Number(this._outputStride),
+    this.model = new PoseNet(this.modelArch, Number(this._version), this._useAtrousConv, Number(this._outputStride),
                              this.scaleInputSize, this._type, this._cacheMap, backend, prefer);
     let start = performance.now();
     result = await this.model.createCompiledModel();
     console.log(`compilation result: ${result}`);
     let elapsed = performance.now() - start;
     console.log(`Compilation time: ${elapsed.toFixed(2)} ms`);
```
Comment (pinzhenx) [inline examples/posenet/utils.js]: The compilation time will be output in the `this.model.createCompiledModel();`. The name of `createCompiledModel()` is quite confusing. And now there is no way to get the compilation time outside.

File: examples/third_party/dat.gui.js (hunk 1 above)
Comment (pinzhenx) [inline examples/third_party/dat.gui.js]: syntax error

File: examples/static/css/style.css [hunk 3]
```diff
@@ -1551,6 +1555,267 @@ button.xclose {
...
+#posenet #my-gui-container.fullscreen ul li select:disabled ~ .select__arrow {
+  border-top-color: rgba(255, 255, 255, 0.1);
+}
+
+#posenet ul li .c {
+  display: inline;
+}
+
+#posenet ul li .c input[type=checkbox] {
+  width: 0px !important;
```
Comment (pinzhenx) [inline examples/static/css/style.css]: Setting checkbox's width to zero doesn't hide it on the Mac (Chrome 71)
screen shot 2019-01-26 at 10 40 37 am

**Comment by pinzhenx** [comment]
https://github.com/intel/webml-polyfill/blob/0320c0ecf24f45f0d626ba27c0815ba4f6d4abb7/examples/posenet/main.js#L136
Changing the `scoreThreshold`, `nmsRadius`, `maxDetections`, `showPose`, `showBoundingBox` will stop the camera predicting. I guess `drawResult()` seems to stop the streaming.

File: examples/posenet/src/helperFunc.js [hunk 4]
```diff
@@ -28,7 +30,8 @@ function getInput(inputElement) {
 
 function getURL(version) {
   let address;
-  const urlBase = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
+  // const urlBase = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
+  const urlBase = '../posenet/model/';
```
Comment (pinzhenx) [inline examples/posenet/src/helperFunc.js]: are we going to host the model files ourselves?

Comment (ibelem) [reply examples/posenet/src/helperFunc.js]: Will continue to use https://storage.googleapis.com/tfjs-models/weights/posenet/ by default.

File: examples/posenet/utils.js (hunk 2 above)
Comment (ibelem) [reply examples/posenet/utils.js]: Removed compilationTime code

File: examples/third_party/dat.gui.js (hunk 1 above)
Comment (ibelem) [reply examples/third_party/dat.gui.js]: Fixed in https://github.com/intel/webml-polyfill/pull/525/commits/20dceaa08524973f958fa60704ba07a44e4028ac

File: examples/static/css/style.css (hunk 3 above)
Comment (ibelem) [reply examples/static/css/style.css]: Fixed in https://github.com/intel/webml-polyfill/pull/525/commits/20dceaa08524973f958fa60704ba07a44e4028ac

File: examples/posenet/src/helperFunc.js (hunk 4 above)
Comment (ibelem) [reply examples/posenet/src/helperFunc.js]: Fixed in https://github.com/intel/webml-polyfill/pull/525/commits/20dceaa08524973f958fa60704ba07a44e4028ac

**Comment by ibelem** [comment]
@pinzhenx "drawResult() seems to stop the streaming" issue was fixed as well. PTAL

**Comment by pinzhenx** [comment]
I really appreciate your great work! Look good to me.

**Comment by ibelem** [comment]
Thanks @pinzhenx for the review, merge it.

**Pull request merged**